manifests/cf-manifest/deployments/040-cf-properties.yml

```diff
@@ -2,9 +2,9 @@ meta:
   environment: ~
   default_quota_definitions:
     default:
-       memory_limit: 10240
-       total_services: 100
-       non_basic_services_allowed: true
+       memory_limit: 2048
+       total_services: 10
+       non_basic_services_allowed: false
```

Comment:
This change isn't described in the commit message. Could you explain it (and add to the commit message)?